### ISSUE TITLE
AWS Security Groups docs missing reference to fixed version

### ISSUE THREAD

**Issue by cobrastarfunkel** [pulumi/registry#3058]
File: [themes/default/content/registry/packages/aws/api-docs/ec2/securitygroup/_index.md](https://www.pulumi.com/registry/packages/aws/api-docs/ec2/securitygroup/)

AWS Security Groups docs reference aws_vpc_security_group_egress(ingress)_rule, which I see is a grab from the terraform docs, but I can't find any other Security Group resources besides the ones under EC2 it says not to use.  Can we get a link to the one we should use?  Assuming the docs for it are in there, because I can't find them anywhere.

It's creating a bunch of the same security group every time I do an up, because like this doc says I guess it's not referencing the id.

This is using yaml

**Comment by cobrastarfunkel** [comment]
Yeah I'll ask on slack, but it's less not knowing what to use and more not finding any other Security Group resource in the docs.  There's no aws.vpc resource I see in the docs which is where I would expect aws_vpc_security_group_egress_rule to be.  I do see the EC2 Security Group Rule but that has the same warning as the main EC2 Security Group.

So this is either a bad copy paste from the Terraform docs and the other resource doesn't exist, or the docs for the resource we should be using are hidden somewhere and should definitely be linked to in the warning.

This warning is on every Security Group resource I can find.
![image](https://github.com/pulumi/registry/assets/35236521/e19304c0-5660-40fa-81a3-b1f9ff939246)

**Comment by toriancrane** [comment]
Hi @cobrastarfunkel ,

Following up on your earlier question, the links to the recommended resources can be found here:

- [aws.vpc.SecurityGroupEgressRule](https://www.pulumi.com/registry/packages/aws/api-docs/vpc/securitygroupegressrule/)
- [aws.vpc.SecurityGroupIngressRule](https://www.pulumi.com/registry/packages/aws/api-docs/vpc/securitygroupingressrule/)

I agree that this should be made clearer in the docs and we are looking into the best way to do so.
